lib/testing/src/lib/protractor/core/models/user.model.ts

```diff
@@ -33,14 +33,14 @@ export class UserModel {
     id: number;
 
     constructor(details: any = {}) {
-        const EMAIL_DOMAIN = browser.params?.testConfig?.projectName ? browser.params.testConfig.projectName : 'alfresco';
+        const EMAIL_DOMAIN = browser.params?.testConfig?.emailDomain ? browser.params.testConfig.emailDomain : 'alfresco.com';
```

Comment:
alfresco.com would also generate bounces.
Can you instead default to example.net?

Reply:
done, thanks